www/asset-manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "main.css": "static/css/main.c17080f1.css",
   "main.css.map": "static/css/main.c17080f1.css.map",
-  "main.js": "static/js/main.17818992.js",
-  "main.js.map": "static/js/main.17818992.js.map",
-  "static\\media\\logo.svg": "static/media/logo.5d5d9eef.svg"
+  "main.js": "static/js/main.483bfedd.js",
+  "main.js.map": "static/js/main.483bfedd.js.map",
+  "static\\media\\logo.svg": "static/media/logo.ee7cd8ed.svg"
```

Comment:
I wonder if all these file changes where the name of an asset is the only difference are meant to be stored in source control or generated on the server.